src/cohost.rs

```diff
@@ -183,6 +183,32 @@ pub struct FollowedProject {
     pub handle: String,
 }
 
+#[derive(Debug, Deserialize)]
+#[allow(non_snake_case)]
+pub struct LikedPostsState {
+    #[serde(rename = "liked-posts-feed")]
+    pub liked_posts_feed: LikedPostsFeed,
```

Comment:
autost’s convention is to use original field names, could you please rename?

Reply:
ah. i see

Reply:
ah. i see. never mind ^w^;;

Reply:
![image](https://github.com/user-attachments/assets/1afee0bf-5d19-4c2b-a7a1-95d626f45f13)

Reply:
no can do - original field has hyphens in it